Makefile

```diff
@@ -54,6 +54,9 @@ ifeq ($(TEST_TYPE),coverage)
 	done
 endif
 
+.PHONY: ci
```

Comment:
Why can't we re-use the existing tag?

I'm sorry, I completely fail to see why (at this stage) we need changes *at all* to coredns' source

Reply:
Chris is trying to get the K8s integration testing out of the main repo and out of the main Travis build, which is what you asked for. I think he would have been happy to leave it all in here, but is trying to make it work in the way we understood you wanted. Maybe you can provide more detail on exactly what you are looking for, so we can stop going back and forth on this and wasting time?